src/database/queries.ts

```diff
@@ -32,7 +32,7 @@ const testColl = collection(db, 'test-col');
 const vendorCollection = collection(db, 'vendors');
 const voucherRangeCollection = collection(db, 'voucher-ranges');
 const voucherCollection = collection(db, 'vouchers');
-const transactionCollection = collection(db, 'transactions');
+const invoiceCollection = collection(db, 'transactions');
```

Comment:
The firebase collection is still called 'transactions' and not invoices. Is there any reason for not changing this?

Reply:
Didn't wanna mess with the backend without Sauhard greenlighting things LOL @sauhardjain should we change that?